person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || parent_permission == true
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] As requested by the previous code reviewer, since `of_age?` is a private method, it needs to be preceded by the `private` keyword like so; 

   `private` 

  `def is_of_age?`
    `@age >= 18`
 `end`. Please kindly fix that.